browser/module.go

```diff
@@ -74,6 +74,27 @@ func (mi *ModuleInstance) Exports() k6modules.Exports {
 
 type mapping map[string]any
 
+// wildcards is a list of extra mappings for our API (api/).
+func wildcards() map[string]string {
```

Comment:
Looks like this and `isWidlCard` are only used in the tests, maybe they could be moved to the test file for now?

Reply:
Resolved in https://github.com/grafana/xk6-browser/pull/673/commits/b298d140f163bcc7594eb303588c631a8bcc40fc.